.gitignore

```diff
@@ -45,3 +45,6 @@ codacy/tmpcharts
 
 # Automated generation of changelogs and release notes
 release-notes-tools
+
+# explicitly ignore the version file
+.version
```

Comment:
The new makefile target creates this file.
We do not want to check it in by accident when creating a release and should therefore ignore it.